burn-compute/tests/dummy/tune/benchmark.rs

```diff
@@ -0,0 +1,62 @@
+use std::marker::PhantomData;
+
+use burn_compute::{
+    server::Handle,
+    tune::{Operation, TuneBenchmark},
+};
+use derive_new::new;
+
+use crate::dummy::{
+    tune::{
+        CacheTestFastOn3Type, CacheTestSlowOn3Type, DummyElementwiseAdditionSlowWrongType,
+        DummyElementwiseAdditionType, DummyElementwiseMultiplicationSlowWrongType,
+        DummyElementwiseMultiplicationType,
+    },
+    DummyClient, DummyKernel, DummyServer,
+};
+
+use super::{AdditionOp, CacheTestOp, MultiplicationOp};
+
+#[derive(new)]
+pub struct DummyBenchmark<'a, O> {
+    client: &'a DummyClient,
```

Comment:
The client is made to be cloned, so I would remove the lifetime here as it adds complexity without much benefit.